Gemfile.lock

```diff
@@ -283,31 +278,33 @@ GEM
       nio4r (~> 2.0)
     raabro (1.4.0)
     racc (1.8.0)
-    rack (3.1.5)
-    rack-protection (4.0.0)
+    rack (2.2.9)
+    rack-contrib (2.5.0)
+      rack (< 4)
+    rack-protection (3.2.0)
       base64 (>= 0.1.0)
-      rack (>= 3.0.0, < 4)
-    rack-session (2.0.0)
-      rack (>= 3.0.0)
+      rack (~> 2.2, >= 2.2.4)
+    rack-session (1.0.2)
+      rack (< 3)
     rack-test (2.1.0)
       rack (>= 1.3)
-    rackup (2.1.0)
-      rack (>= 3)
-      webrick (~> 1.8)
-    rails (7.1.3.4)
-      actioncable (= 7.1.3.4)
-      actionmailbox (= 7.1.3.4)
-      actionmailer (= 7.1.3.4)
-      actionpack (= 7.1.3.4)
-      actiontext (= 7.1.3.4)
-      actionview (= 7.1.3.4)
-      activejob (= 7.1.3.4)
-      activemodel (= 7.1.3.4)
-      activerecord (= 7.1.3.4)
-      activestorage (= 7.1.3.4)
-      activesupport (= 7.1.3.4)
+    rackup (1.0.0)
+      rack (< 3)
+      webrick
+    rails (7.2.0)
```

Comment:
Why upgrading Rails?

Reply:
Sorry, I have no idea why I did that 🙏 
Returned the Rails version with https://github.com/kaigionrails/conference-app/pull/256/commits/5da601ca61bd80943689286a9b582084e765d9d4